contracts/DHTT.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.4.18;
+
+import './BaseContracts.sol';
+
+
+contract DHTT is MintableToken {
+    
+    string public constant name = "go to https://try.dehedge.com";
+    
+    string public constant symbol = "DHTT";
+    
+    uint32 public constant decimals = 0;
+    
+    function DHTTCoin() public {        
+        totalSupply = 3000000;
+       balances[owner] = totalSupply;
+    }
+    
+    function transferMany(uint256 _value, address[] receivers) onlyOwner public returns (bool) {
```

Comment:
what you think about checking the length of `receivers` to avoid long arrays and gas limit hit?